src/api/app/controllers/request_controller.rb

```diff
@@ -156,7 +156,7 @@ def request_create
 
     # cache the diff (in the backend)
     @req.bs_request_actions.each do |a|
-      a.delay.webui_infos
+      BsRequestActionWebuiInfosJob.perform_later(a.id)
```

Comment:
depends on your definition of quick - this is creating source diffs. While @mlschroe optimized it, it can still take minutes :)